packages/ingest/HttpProviderClient.js

```diff
@@ -19,20 +19,25 @@ const log = require('@cumulus/common/log');
 const isValidHostname = require('is-valid-hostname');
 const { buildURL } = require('@cumulus/common/URLUtils');
 const errors = require('@cumulus/errors');
+const get = require('lodash/get');
+
 const { lookupMimeType, decrypt } = require('./util');
 
 const validateHost = (host) => {
   if (isValidHostname(host) || isIp(host)) return;
 
   throw new TypeError(`provider.host is not a valid hostname or IP: ${host}`);
 };
+const HTTP_REQUEST_TIMEOUT_SECONDS = 5 * 60; // Timeout for simplecrawler's headers response
 
 class HttpProviderClient {
   constructor(providerConfig) {
     this.providerConfig = providerConfig;
     this.protocol = providerConfig.protocol;
     this.host = providerConfig.host;
     this.port = providerConfig.port;
+    this.httpListTimeout = get(providerConfig,
```

Comment:
Based on [this conversation](https://github.com/nasa/cumulus/pull/1755#discussion_r452460205), I think I agree. Let's remove the default `HTTP_REQUEST_TIMEOUT_SECONDS` override from this file. If they pass in an `httpListTimeout` value, then use that to override `c.timeout`. Otherwise, use the crawler's default.

Reply:
a81e2c793